Client/Frontend/Browser/Tabs/RemoteTabsPanel.swift

```diff
@@ -155,28 +180,41 @@ class RemoteTabsPanelClientAndTabsDataSource: NSObject, RemoteTabsPanelDataSourc
         * Ideally, we should save and use the modified time of the tabs record itself.
         * This will be the real time that the other client uploaded tabs.
         */
-
-        let image: UIImage?
-        if client.type == "desktop" {
-            image = UIImage.templateImageNamed("deviceTypeDesktop")
-            image?.accessibilityLabel = .RemoteTabComputerAccessibilityLabel
-        } else {
-            image = UIImage.templateImageNamed("deviceTypeMobile")
-            image?.accessibilityLabel = .RemoteTabMobileAccessibilityLabel
-        }
         return view
     }
 
-    fileprivate func tabAtIndexPath(_ indexPath: IndexPath) -> RemoteTab {
+    func tabAtIndexPath(_ indexPath: IndexPath) -> RemoteTab {
         return clientAndTabs[indexPath.section].tabs[indexPath.item]
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: RemoteTabIdentifier, for: indexPath) as! SimpleTwoLineCell
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: TwoLineImageOverlayCell.cellIdentifier,
+                                                       for: indexPath) as? TwoLineImageOverlayCell
+        else {
+            return UITableViewCell()
+        }
+
         let tab = tabAtIndexPath(indexPath)
         cell.titleLabel.text = tab.title
         cell.descriptionLabel.text = tab.URL.absoluteString
-        // TODO: Bug 1144765 - Populate image with cached favicons.
+
+        cell.leftImageView.layer.borderColor = UX.IconBorderColor.cgColor
+        cell.leftImageView.layer.borderWidth = UX.IconBorderWidth
+        cell.accessoryView = nil
+
+        // TODO: Load favicon image from remote tab (tab.faviconURL) https://mozilla-hub.atlassian.net/browse/FXIOS-3754
+        // Client image is temporary and should be removed once we have the favicon URL
+        // siteImageHelper is already in the class to load from URL, just need to uncomment code and adjust
+        let client = self.clientAndTabs[indexPath.section].client
+        let image = getClientImage(client: client)
+        cell.leftImageView.image = image
+        cell.leftImageView.backgroundColor = UIColor.theme.general.faviconBackground
+
+//        getFavIcon(for: tab) { [weak cell] image in
+//            cell.leftImageView.image = image
+//            cell.leftImageView.backgroundColor = UIColor.theme.general.faviconBackground
+//        }
+
```

Comment:
All this section here is specifically for https://mozilla-hub.atlassian.net/browse/FXIOS-3754
Has temp solution in place, but code is ready when we have the favicon URL